src/jsSSE.php

```diff
@@ -107,6 +113,10 @@ private function output(string $content): void
      */
     public function sendBlock(string $id, string $data, string $name = null): void
     {
+        if (connection_aborted()) {
```

Comment:
Very important here is to check for `closeBeforeUnload` too, otherwise we should keep the SSE process running even if the connection between browser was interrupted.

Reply:
` ignore_user_abort(true); ` in jsSSE::initSse() should take care of that. 
It will keep the script running, event if user aborted it.

Reply:
Yes, but then this will call exit - and a lot of SSE tasks may rely on fact that they should normally always finish no matter if the connections/browser is still active or not.

Reply:
Done